### PR TITLE
docs: tags: streamline indentation

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
streamline indentation a bit here.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by gundalow — APPROVED** [review]
Thanks!

**Comment by gundalow** [comment]
rebuild_merge

File: docs/docsite/rst/user_guide/playbooks_tags.rst [hunk 1]
```diff
@@ -154,9 +154,9 @@ Playbook file::
 
     - hosts: all
       tasks:
-      - include_role:
-          name: myrole
-        tags: mytag
+        - include_role:
```
Comment (dagwieers) [inline docs/docsite/rst/user_guide/playbooks_tags.rst]: Something we have to discuss in the Documentation WG to have consistent examples.

My preference is to indent minimally (so only the amount required by YAML).
Just like I recommend quoting only when it is necessary.
This gives users a better feeling to how YAML parses files and when something is mandatory.

**Review by dagwieers — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/user_guide/playbooks_tags.rst (hunk 1 above)
Comment (resmo) [reply docs/docsite/rst/user_guide/playbooks_tags.rst]: +1

**Comment by resmo** [comment]
ready_for_review

**Review by dagwieers — APPROVED** [review]
I personally like this better. But usually the consensus is something else anyway ;-)

File: docs/docsite/rst/user_guide/playbooks_tags.rst (hunk 1 above)
Comment (acozine) [reply docs/docsite/rst/user_guide/playbooks_tags.rst]: +1 to fixing indentation, though I don't think it's our highest priority

**Comment by acozine** [comment]
This is an improvement, so I'll merge. I'd like to see a `name` line on every task in our user-facing documentation as well, but that can be a future PR.

**Comment by dagwieers** [comment]
@acozine I agree, but for newer contributions we could already lay out what we would like to see.
I prefer we don't have to have too many rules though and rather not enforce this, but instead bulk-fix it before a new release ?

**Pull request merged**